docs/user_guide/preemption.md

```diff
@@ -0,0 +1,252 @@
+
+For example, even with `allow-preemption` set to `false`, DaemonSet pods can still be preempted. Additionally, if an application in `queue-1` has a higher priority than one in `queue-3`, the application in `queue-2` will be preempted because an application can never preempt another application with a higher priority. In such cases where no other preemption options exist, the `allow-preemption` flag may not prevent preemption.
+
+
+| Queue                  | Resource before preemption | Resource after preemption              |
+| ---------------------- | -------------------------- | -------------------------------------- |
+| `priority-class.queue-1` | 8 (victim)                 | 5                                      |
+| `priority-class.queue-2` | 8                          | 8                                      |
+| `priority-class.queue-3` | 0                          | 3 (guaranteed resources are satisfied) |
+
+![preemption_priorityclass_case](../assets/preemption_priorityclass_case.png)
+
+### Priority Queue
+
+In addition to utilizing the default PriorityClass in Kubernetes, you can configure priorities directly on a [YuniKorn queue](priorities).
+
+In the following example, we will demonstrate preemption based on queue priority.
+
+We will deploy five pods with a resource demand of 3 in the `high-priority` queue, `normal-priority` queue, and `low-priority` queue, respectively. We will deploy them to the `normal-priority` queue first, ensuring that the resources in the `priority-queue`(parent queue) will be fully utilized. This will result in uneven resource distribution among the queues, triggering preemption.
+
+| Queue                            | Max Resource | Guaranteed Resource | priority.offset |
+| -------------------------------- | ------------ | ------------------- | --------------- |
+| `priority-queue`                 | 18           | -                   |                 |
+| `priority-queue.high-priority`   | 10           | 6                   | 100             |
+| `priority-queue.normal-priority` | 18           | 6                   | 0               |
+| `priority-queue.low-priority`    | 10           | 6                   | -100            |
+
+Result:
+
+A queue with higher priority can preempt resources from a queue with lower priority, and preemption stops when the queue has preempted enough resources to satisfy its guaranteed resources.
+
+| Queue                            | Resource before preemption | Resource after preemption              |
+| -------------------------------- | -------------------------- | -------------------------------------- |
+| `priority-queue.high-priority`   | 0                          | 6 (guaranteed resources are satisfied) |
```

Comment:
Formatting again - use `(guaranteed minimum)`. Also, shorten queue names to fit without wrapping.